.github/workflows/build.yml

```diff
@@ -19,6 +19,27 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: '15.x'
+
+      - uses: actions/cache@v2
+        id: pip-cache
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
```

Comment:
Thank you for figuring this stuff out.  I never did manage to find the time to figure out caching.

Reply:
Actually this is just copy and paste of the examples from https://github.com/actions/cache/blob/main/examples.md#node---yarn

But it does seem to work :)